api/scripts/bigint/answer/add-constraints-and-indexes-to-answers-ke-temporary-tables-with-bigint.js

```diff
@@ -0,0 +1,164 @@
+    await knex.raw('ALTER TABLE "knowledge-elements_bigint" ALTER COLUMN "earnedPix" SET NOT NULL').transacting(trx);
+
+    performance.mark('keBigIntNotNullConstraintsTime');
+    performance.measure(
+      'keBigIntNotNullConstraints',
+      'answersBigintAssessmentidIndex',
+      'keBigIntNotNullConstraintsTime'
+    );
+
+    await knex
+      .raw('ALTER TABLE "knowledge-elements_bigint" ADD CONSTRAINT "knowledge-elements_bigint_pkey" PRIMARY KEY("id")')
+      .transacting(trx);
+
+    performance.mark('keBigintPkey');
+    performance.measure('knowledge-elements_bigint', 'keBigIntNotNullConstraintsTime', 'keBigintPkey');
+
+    await knex
+      .raw(
+        `ALTER TABLE "knowledge-elements_bigint"
+           ADD CONSTRAINT "knowledge_elements_bigint_answerid_foreign" FOREIGN KEY("answerId")
+           REFERENCES "answers_bigint"(id)`
+      )
+      .transacting(trx);
+
+    performance.mark('keBigintAnsweridForeign');
+    performance.measure('knowledge_elements_bigint_answerid_foreign', 'keBigintPkey', 'keBigintAnsweridForeign');
+
+    await knex
+      .raw(
+        `ALTER TABLE "knowledge-elements_bigint"
+           ADD CONSTRAINT "knowledge_elements_bigint_assessmentid_foreign" FOREIGN KEY ("assessmentId")
+           REFERENCES "assessments" (id)`
+      )
+      .transacting(trx);
+
+    performance.mark('keBigintAssessmentidForeign');
+    performance.measure(
+      'knowledge_elements_bigint_assessmentid_foreign',
+      'keBigintAnsweridForeign',
+      'keBigintAssessmentidForeign'
+    );
+
+    await knex
+      .raw(
+        `ALTER TABLE "knowledge-elements_bigint"
+           ADD CONSTRAINT "knowledge_elements_bigint_userid_foreign" FOREIGN KEY ("userId")
+           REFERENCES "users" (id)`
+      )
+      .transacting(trx);
+
+    performance.mark('keBigintUseridForeign');
+    performance.measure(
+      'knowledge_elements_bigint_userid_foreign',
+      'keBigintAssessmentidForeign',
+      'keBigintUseridForeign'
+    );
+
+    await knex
+      .raw(
+        'CREATE INDEX "knowledge_elements_bigint_userid_index" ON "knowledge-elements_bigint" USING btree ("userId")'
+      )
+      .transacting(trx);
+
+    performance.mark('keBigintUseridIndex');
+    performance.measure('knowledge_elements_bigint_userid_index', 'keBigintUseridForeign', 'keBigintUseridIndex');
+  });
+
+  performance.mark('endTime');
+  performance.measure('Global time for adding constraints and indexes', 'startTime', 'endTime');
+
+  logger.debug('Finish adding constraints and indexes to answers ke bigint tables');
+
+  performance.clearMarks();
+  obs.disconnect();
+};
+
+const isLaunchedFromCommandLine = require.main === module;
+
+async function main() {
+  try {
+    logger.debug('Start script... ');
+    await addConstraintsAndIndexesToAnswersKeBigintTemporaryTables();
+    logger.debug('End script: copy done successfully.');
```

Comment:
On peut au moins mettre ça par défaut
```suggestion
    logger.info('Start script... ');
    await addConstraintsAndIndexesToAnswersKeBigintTemporaryTables();
    logger.info('End script: copy done successfully.');
```